scripts/autorelease.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p gitAndTools.hub git -i bash
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# Project name, inferred from repository name
+project=$(basename $(pwd))
+
+# The directory in which artifacts will be created
+TEMPDIR=`mktemp -d`
+
+# Build release.nix
+nix-build release.nix -o $TEMPDIR/$project --arg timestamp $(date +\"%Y%m%d%H%M\")
+
+# Delete release
+hub release delete auto-release
+
+# Update the tag
+git fetch # So that the script can be run from an arbitrary checkout
+git tag -f auto-release
+git push --force --tags
+
+# Combine all assets
+assets=""
```

Comment:
Why not create a tarball with all the assets?

Reply:
Because customers want to be able to have direct links to the binaries:
https://github.com/serokell/tezos-packaging/issues/11#issuecomment-571239514

Reply:
Moreover, the number of assets is going to increase soon, and end-users are likely will not want to download all binaries/packages